code/game/gamemodes/wizard/spellbook.dm

```diff
@@ -496,6 +496,14 @@
 	log_name = "SI"
 	category = "Weapons and Armors"
 
+/datum/spellbook_entry/item/spell_blade
```

Comment:
```suggestion
/datum/spellbook_entry/item/cursed_katana
```
this smells of untested code >:| (unless this somehow works in its current state?)

Reply:
Whoops sorry